### PR TITLE
[full-ci] Fix right sidebar content on small screens

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
We've fixed the right sidebar content on small screens because some screen sizes caused the content to flow out of the screen. Things that have been done to achieve this:

* Selection info has been removed.
* Labels of the batch actions will hide on screens <1280px if the sidebar is open.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/7498

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

**Comment by JammingBen** [comment]
@kulmann How "bad" is it that the batch actions always show a tooltip if the right sidebar is open? If that's an issue, I probably need to solve this in JS and use a resize observer... which I _really_ don't want to 😅

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/27765/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/27765/tracing/file-alice-2022-8-23-01-56-26.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/27765/tracing/file-brian-2022-8-23-01-56-32.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/27765/tracing/folder-alice-2022-8-23-01-54-57.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/27765/tracing/folder-brian-2022-8-23-01-55-04.zip 


</p></details>

File: packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue [hunk 1]
```diff
@@ -84,5 +87,14 @@ export default {
     display: flex;
     gap: var(--oc-space-small);
   }
+
+  &-squashed {
```
Comment (kulmann) [inline packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue:91]: Why only for squashed state? IMO we could do the same for when the sidebar is closed.

File: packages/web-app-files/src/components/AppBar/AppBar.vue [hunk 2]
```diff
@@ -32,13 +32,18 @@
         </div>
       </div>
       <div class="files-app-bar-actions">
-        <div class="oc-flex-1 oc-flex oc-flex-start" style="gap: 15px">
+        <div class="oc-flex-1 oc-flex oc-flex-start">
           <slot v-if="showActionsOnSelection || selectedFiles.length === 0" name="actions" />
-          <size-info
+          <oc-button
```
Comment (kulmann) [inline packages/web-app-files/src/components/AppBar/AppBar.vue:37]: We could easily implement the `Clear selection` action as a file action (see actions mixins). That would remove the need for special code here. What do you think?

Comment (JammingBen) [reply packages/web-app-files/src/components/AppBar/AppBar.vue:37]: Good idea, I'll do it that way 👍

**Comment by kulmann** [comment]
I have one more: with an active file clipboard we'd need to apply the same logic to buttons without selection. You can reach this state by e.g. copying or cutting files and then navigating into a different folder. See screenshot below, on small screens the right sidebar gets cropped again. For the `Paste here` button I suspect that we'll need a divider between the `Paste` and `Cancel` icons in the button group...

Screenshot 2022-08-24 at 14 47 01

**Comment by JammingBen** [comment]
Oof. Yeah that's right, haven't thought of that. The same applies to the "New" and the "Upload" button as well, but only in the "active clipboard" + sidebar open state, correct?

**Comment by kulmann** [comment]
> Oof. Yeah that's right, haven't thought of that. The same applies to the "New" and the "Upload" button as well, but only in the "active clipboard" + sidebar open state, correct?

Thought about it with @tbsbdr - could we hide the labels of `New` and `Upload`, but keep the label of `Paste here`? The first two buttons are easier to understand than the paste button.

**Comment by JammingBen** [comment]
Sure. One question remains though: How do we deal with tooltips? The batch actions now always have a tooltip - the same would apply here. The alternative would be to mess around with a resize observer and toggle the tooltips conditionally depending on the screen size.

**Comment by tbsbdr** [comment]
would be good to hide the tooltips if the labels are visible

File: packages/web-app-files/src/components/AppBar/AppBar.vue [hunk 3]
```diff
@@ -79,6 +76,12 @@ export default {
     hasViewOptions: { type: Boolean, default: true },
     showActionsOnSelection: { type: Boolean, default: false }
   },
+  data: function () {
+    return {
+      resizeObserver: new ResizeObserver(this.onResize),
+      limitedScreenSpace: false
```
Comment (JammingBen) [inline packages/web-app-files/src/components/AppBar/AppBar.vue:82]: Bit of an ugly name. I had set it to "showTooltips" originally, but the TopBar component should not determine wether to show or hide tooltips - that's up to the component itself. Therefore the TopBar only provides the info that limited screen space is available.

Comment (kulmann) [reply packages/web-app-files/src/components/AppBar/AppBar.vue:82]: Isn't the same valid for the `BatchActions` component then? That one still has the `showTooltips` prop.

**Review by kulmann — APPROVED** [review]

File: packages/web-app-files/src/components/AppBar/AppBar.vue (hunk 3 above)
Comment (JammingBen) [reply packages/web-app-files/src/components/AppBar/AppBar.vue:82]: For `BatchActions`, there is no other condition to show tooltips. Hence I figured this component can have a `showTooltips` prop.

**Pull request merged**